content/ui-patterns/feature-onboarding.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: Feature onboarding
+---
+
+import {Box, Flex, BorderBox, Text} from '@primer/components'
+
+This is a guide for designing onboarding for the product and does not include what to do for marketing pages, email announcements, social media, etc.
+
+Onboarding is a virtual unboxing experience that helps users get started with an app.
+
+- ✅ Think creatively utilizing different ways to onboard users, such as empty states and off-product avenues.
+- ✅ For feature annoucements, prepare a release timeline to call out new features.
+- 🚫 Do not leave feature annoucement campaigns running forever.
+
+# Guiding principles
+## Proximity and proportion
+Onboarding elements should maintain an appropriate proximity to where the feature will live in perpetuity.
+
+Consider the primary job to be done on a page before disrupting the user with a feature callout.  Keep in mind the context, placement, and dominance of an onboarding element. Onboarding UI choices should be proportional to the hierarchy of the feature on the page.
```

Comment:
The 4 sentences in this section could be a bulleted list. When you say "Keep in mind the context, etc." what do you mean exactly, as a guideline?

Reply:
> When you say "Keep in mind the context, etc." what do you mean exactly, as a guideline?
@ohjoycelau any ideas?

Reply:
Probably something like:

"Be aware of the loudness of a pattern; use patterns that are appropriate to it's functional hierarchy on a page."

i.e. we often see new features that get these loud callouts (popovers) when they are, honestly, often features that are too infrequently used or not a core function of the page.